### PR TITLE
Metropolitan office change to Met Office

### PR DESCRIPTION
**Pull request by corinnebosley**

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Support
 _______
 
 GeoViews was developed through a collaboration between
-`Continuum Analytics <https://continuum.io>`_ and the `UK Metropolitan Office
+`Continuum Analytics <https://continuum.io>`_ and the `UK Met Office
```
Comment (rhattersley) [inline doc/index.rst]: I think the brand people prefer just "Met Office" (without the UK prefix).

**Comment by philippjfr** [comment]
Didn't you do this once already? Sorry about that, seems like I didn't include your change before when moving it. Will merge now.

**Comment by rhattersley** [comment]
> I think the brand people prefer just "Met Office" (without the UK prefix).

:unamused:

**Comment by corinnebosley** [comment]
Sorry Richard, was just about to push and then Philip merged it.  You want me to kick off?

**Pull request merged**